jobrunner/record_stats.py

```diff
@@ -34,7 +37,11 @@ def main():
         return
     log.info(f"Logging system stats to: {database_file}")
     connection = get_database_connection(database_file)
+    last_run = None
```

Comment:
It looks like last_run isn't being set to something other than None, so record_tick_trace isn't going to get called?

Reply:
Thanks, good catch. A missing initialisation clause, fix incoming